WcaOnRails/app/models/timestamp.rb

```diff
@@ -2,4 +2,8 @@
 
 class Timestamp < ApplicationRecord
   self.primary_key = "name"
+
+  def not_after?(other_date)
```

Comment:
Why call this not_after instead of before? Something to do will nil? Like nil is not after and it's not before?